services/integration.sh

```diff
@@ -163,7 +163,7 @@ while [ "$all_services_are_up" == "" ]; do
 done
 echo "all services are up!"
 
-( ${EXE} "${@:3}" && echo 0 > "${EXIT_STATUS_LOCATION}" && kill_gracefully ) || kill_gracefully &
+( ${EXE} "${@:2}" && echo 0 > "${EXIT_STATUS_LOCATION}" && kill_gracefully ) || kill_gracefully &
```

Comment:
now i'm confused.  shouldn't this actually be 4 rather than the 3 i put there?  since we are getting two new arguments sometimes, namely `-b` and `../brig/brig.integration.yaml`?

Reply:
I stumbled accross while debugging argument passing to `spar`.

`${@:3}` contains the args starting from the third element, which in our case is (`echo ${EXE} "${@:3}"` )
```
../../dist/spar-integration spar.integration.yaml -i ../integration.yaml -b ../brig/brig.integration.yaml
```
Note how the `-s` parameter is dropped.
This however doesn't result in incorrect arguments to `spar-integration`, because it doesn't feed the arguments verbatim to the argument parser, but splits them (see `services/spar/test-integration/Spec.hs:55`):
```
wireArgs: ["-i","../integration.yaml","-b","../brig/brig.integration.yaml"]
hspecArgs: ["spar.integration.yaml"]
```

With `${@:2}` all arguments are passed to `spar-integration`
```
../../dist/spar-integration -s spar.integration.yaml -i ../integration.yaml -b ../brig/brig.integration.yaml
```
resulting in
```
wireArgs["-s","spar.integration.yaml","-i","../integration.yaml","-b","../brig/brig.integration.yaml"]
hspecArgs[]
```

Both version (`2` and `3`) seem to work (at least for `spar-integration`) .

We should probably revert this commit, because this might break all other integration test.
But I really want to understand what's going on here.